plasmapy/physics/parameters.py

```diff
@@ -370,9 +374,13 @@ def thermal_speed(T, particle="e-", method="most_probable"):
         The particle temperature is invalid or particle cannot be used to
         identify an isotope or particle
 
-    UserWarning
-        If the particle thermal speed exceeds 10% of the speed of light, or
-        if units are not provided and SI units are assumed.
+    Warns
+    -----
+    RelativityWarning
+        If the ion sound speed exceeds 10% of the speed of light, or
```

Comment:
Incomplete sentence?

Reply:
Yup. My bad, that was some quick dirty work.